### ISSUE TITLE
add draft check command

### ISSUE THREAD

**Issue by michelleN** [Azure/draft#910]
like the `linkerd check` command, ensuring all systems go

**Comment by bjornmagnusson** [comment]
God yes! I´m so impressed by how it works in Linkerd.
I would be down to help out with this.
What are your thoughts about this for draft? 
check that k8s api / draft config setup / helm is available on working? 
or completely different? :)

**Comment by michelleN** [comment]
yes exactly!! Generally for features, we aim to write a [Draft Enhancement Proposal/DEP](https://github.com/Azure/draft/tree/master/docs/reference) before starting so everyone is on the same page about the feature. If you could take a stab at writing a DEP first, that would be awesome. If we don't want it to be top-level, we can also make a draft check plugin first. Either approach is fine in my opinion but we'll need buy-in from the other core maintainers to make this a top level command first. 
cc/ @fibonacci1729 @bacongobbler @radu-matei 

Thank you!

**Comment by bacongobbler** [comment]
I'd love to have an "all systems nominal and responding" check command baked right into the tool. A DEP would be great so we can figure out what parts of the system we need to check and how that'd look from a UX perspective. That way we can also explain what purpose `draft check` intends to solve (and more importantly what it does *not* intend to solve).

Let us know if you need any help!

**Comment by bjornmagnusson** [comment]
Sure thing, will do that. Thanks for the pointer. Makes sense
